server/services/store/sqlstore/migrate.go

```diff
@@ -145,9 +158,15 @@ func (s *SQLStore) Migrate() error {
 	if err != nil {
 		return err
 	}
+
+	fmt.Println("############################################")
```

Comment:
This looks like debug block. I think you can remove this.

Reply:
Done.